### DISCUSSION TITLE
How to use Presenter view

### DISCUSSION THREAD

**Discussion by chenxizhang** [marp-team/marp#240] [Q&A] [answered]
First of all, huge thanks to Marp team, you guys doing a great job indeed.

I am wondering what the "Presenter view" exactly mean? When I used PowerPoint, I know that I can turn on the presenter view, then I can see the note of each slide, and others can not.

Is there anyway to implement this scenario with Marp?

**Comment by yhatt** [comment, marked as answer]
In Marp, "Presenter view" means a exactly same feature as in famous presentation tools. If exported Marp slides to HTML, you should be able to open new presenter view window from the prompter icon.

Example: https://yhatt.github.io/marp-cli-example/?view=presenter

**Reply by chenxizhang** [reply]
Thanks, Yuki. 

I am trying to understand how it will work. But it seems like there is also a bug related to the presenter view. You can try this link https://graph.xizhang.com ， when the page loaded, you try to open it in presenter view, you will see a interesting thing there, just as below. 

![image](https://user-images.githubusercontent.com/1996954/146874041-af33ddfa-3b6e-4bff-b48a-3f1acea6897e.png)

but if you navigate to page 2, you will see the page 2 is not same of page 1.

**Reply by chenxizhang** [reply]
so, if I turned on the presenter view, and the computer was connected to a projector, then I will see the presenter view in my desktop, but the audiences will just see the page like below? 

I just not have projector to test this scenario, but if it works , that will be a great thing, it's amazing.

![image](https://user-images.githubusercontent.com/1996954/146874348-45602dc7-d263-4f34-959b-165eca323fc6.png)

**Reply by chenxizhang** [reply]
I think I've understand the scenario now. when I have two screens, I can leave the original page view in one screen, and then turned on the presenter view, in another screen, and they will be synced when I navigating between the pages. 

Oh, although it introduced a little bit confusing, but it's a very nice design. and it works for me, will help me to do the online presentation then.


![96540aa6eaa5476aebfd88f7bd38f00](https://user-images.githubusercontent.com/1996954/146876031-1229d878-3e3e-4fcc-b1c4-d3c76e66f649.jpg)
